yang/frr-isisd.yang

```diff
@@ -179,7 +179,6 @@ module frr-isisd {
   grouping redistribute-attributes {
     description
       "Common optional attributes of any redistribute entry.";
-    choice attribute {
```

Comment:
Please reindent all nested nodes accordingly (`s/^  //`).

Reply:
>     * Please change the commit message of your first commit to describe what problem it's solving;
 
Need create common describe for all issues?

Reply:
Only the first commit, the commit message should explain why the change is necessary (unless it's a trivial change, which isn't case here).